7_react/react2/week1/hyf/src/components/InfoPanel.js

```diff
@@ -0,0 +1,15 @@
+import "./InfoPanel.css";
+
+export default function InfoPanel(props) {
+  return (
+    <>
+      <div className="container">
+        <img className="my-img" src={props.imageUrl}></img>
+        <div className="content">
+          <h2 className="my-title">{props.title}</h2>
+          {props.children}
+        </div>
+      </div>
+    </>
+  );
```

Comment:
Here we used the `props.children` to tell React where we want the custom HTML to appear. In our case we placed it underneath out heading.